apps/discussions/package.json

```diff
@@ -1,19 +1,20 @@
 {
   "name": "@tps/apps-discussions",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "",
   "dependencies": {
     "@aragon/api": "latest",
```

Comment:
Since we are moving some dependencies around, it should be a good chance to fix this, that could break something if we don't control which version are we using for `@aragon/api` and `@aragon/api-react`
Shouldn't be better to use a more specific version to have more control over it?